Marlin/Configuration.h

```diff
@@ -1753,6 +1753,9 @@
  * View the current statistics with M78.
  */
 //#define PRINTCOUNTER
+#if ENABLED(PRINTCOUNTER)
+  #define PRINTCOUNTER_SAVE_INTERVAL 1   // (seconds) EEPROM save interval during print
```

Comment:
I would be careful with this as 1 sec would damage some eeprom within a couple of days. Maybe a warning for low values would help?